cmd/commands/destroy_cluster.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/crds"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+// DestroyClusterCmd represents the command for destroy cluster
+var DestroyClusterCmd = &cobra.Command{
+	Use:   "destroy-cluster",
+	Short: "delete all Rook CRDs",
+	Args:  cobra.NoArgs,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		destroyClusterConfirmation, _ := cmd.Flags().GetBool("yes-really-destroy-cluster")
+		if !destroyClusterConfirmation {
+			fmt.Println("Error: you need to provide the flag --yes-really-destroy-cluster to confirm the cluster deletion")
```

Comment:
```suggestion
			logging.Error("you need to provide the flag --yes-really-destroy-cluster to confirm the cluster deletion")
```

Reply:
Instead of a flag, let's follow the pattern we have a couple other places where we prompt "Are you sure you want to destroy the cluster in namespace %s?". Then they have to type "yes-really-destroy-cluster", unless they have set the env var to skip prompts.